src/shared/logger/winstonToolkitLogger.ts

```diff
@@ -119,11 +119,18 @@ export class WinstonToolkitLogger implements Logger, vscode.Disposable {
     }
 
     private mapError(level: LogLevel, err: Error): Error | string {
+        // Use ToolkitError.trace even if we have source mapping (see below), because:
+        // 1. it is what users will see, we want visibility into that when debugging
+        // 2. it is often more useful than the stacktrace anyway
+        if (err instanceof ToolkitError) {
+            return err.trace
+        }
+
         if (isSourceMappingAvailable() && level === 'error') {
```

Comment:
@JadenSimon I changed the logic to prefer ToolkitError.trace because (1) it's closer to what the user will see (which is useful for deciding if a failure provides enough info) and (2) the stacktrace isn't that useful.

Reply:
Yeah the native stack trace also doesn't show all exceptions which makes it even less useful. Hopefully that won't be as much of an issue when 'cause' is added to native ES exceptions.